packages/server/graphql/mutations/createPoll.ts

```diff
@@ -116,7 +116,7 @@ const createPoll = {
         teamId
       }
     })
-    publish(SubscriptionChannel.MEETING, meetingId, 'AddPollSuccess', data, subOptions)
+    publish(SubscriptionChannel.MEETING, meetingId, 'CreatePollSuccess', data, subOptions)
```

Comment:
Meeting subscription was crashing because of that, I probably missed that when implementing backend 🕵️‍♂️